staging/src/k8s.io/client-go/tools/clientcmd/client_config_test.go

```diff
@@ -526,3 +527,43 @@ func TestNamespaceOverride(t *testing.T) {
 
 	matchStringArg("foo", ns, t)
 }
+
+func TestNamespacePrefixStrip(t *testing.T) {
```

Comment:
Maybe the two test funcs can be in one single test func with multiple test cases. Anyway, this is ok as-is.